elm/mldataset/wrap_sklearn.py

```diff
@@ -64,38 +67,46 @@ class SklearnMixin:
     _as_numpy_arrs = _as_numpy_arrs
     _from_numpy_arrs = _from_numpy_arrs
 
-    def _call_sk_method(self, sk_method, X=None, y=None, **kw):
+    def _call_sk_method(self, sk_method, X=None, y=None, do_split=True, **kw):
```

Comment:
I am currently working on simplifying this function - checking what is actually needed.